.env

```diff
@@ -0,0 +1,31 @@
+# In all environments, the following files are loaded if they exist,
```

Comment:
`.env` should not be committed to the repo

Reply:
@rasmuswinter this is what I do by default too, but it's not the Symfony convention now. https://symfony.com/blog/improvements-to-the-handling-of-env-files-for-all-symfony-versions

If we diverge we will lose the auto management of this by Flex for e.g. new vars.